block_element.go

```diff
@@ -171,12 +171,6 @@ func (s *ButtonBlockElement) WithStyle(style Style) *ButtonBlockElement {
 	return s
 }
 
-// WithConfirm adds a confirmation dialogue to the button object and returns the modified ButtonBlockElement
-func (s *ButtonBlockElement) WithConfirm(confirm *ConfirmationBlockObject) *ButtonBlockElement {
```

Comment:
we're losing this - sure we don't want it?

Reply:
We're adding this, no? If you look at [`e56f38b` (#12)](https://github.com/incident-io/slack/pull/12/commits/e56f38bc7dacfdc521d59d842325198983524612#diff-53b57cd5be231077bacd61bb89ad77ef9bdd61ae4a6eced4a785f434bf508e16R174-R179)